### DISCUSSION TITLE
component with q-dialog not showing in Vue devtools with Vue3

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#10827] [General - Components / Directives / etc]
Since upgrading to Q2, Vue devtools is no longer able to show dialog components. Is there a way to get this capability back?

**Comment by MilosPaunovic** [comment]
Have you installed beta version with support for `Vue3`?
https://chrome.google.com/webstore/detail/vuejs-devtools/ljjemllljcmogpfapbkkighbhhppjdbg

**Reply by CrawfordW** [reply]
Yes I have. Works fine for main components, but dialogs no longer show up in the component list.

**Reply by ontwikkelfabriek** [reply]
@CrawfordW 

>  but dialogs no longer show up in the component list.

With dialogs I presume  you mean Quasar's dialog component and not Vue3 dialogs in general?

If so then please create an issue for this.

**Reply by CrawfordW** [reply]
OK will do.

**Comment by hawkeye64** [comment]
I think the way it works is that new Vue Dev Tools, just look at everything inside the Vue-mounted area. I also think that Quasar dialogs (using the plugin), puts the HTML at the document level, which is outside of where VDT is looking (it used to look at the whole document). I believe this includes qmenu and qtooltip components.

**Reply by CrawfordW** [reply]
I created an issue which was answered. The trick is to use the new location icon in devtools then click on the desired component to see it's content. Works great!
